apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -546,6 +624,60 @@ public Response dexEthInfo(@Context SecurityContext securityContext) throws NotF
         }
     }
 
+    
+        
+    @Path("/flood")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "test flooding",
+            description = "initiates flooding table with the test data",
+            tags = {"debug"},
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "Successful execution",
+                            content = @Content(mediaType = "application/json",
+                                    schema = @Schema(implementation = ResponseDone.class)))
+            }
+    ) 
+    public ResponseDone startTestFlood(  ) {       
```

Comment:
Maybe create ordinary api tests and service tests

Reply:
Relocated it from there to unit tests, thanks.